### PR TITLE
Improvements for ARM64

### PR DESCRIPTION
**Pull request by PhilDevProg**
- Add janus config files for arm64 and use it after checking for architecture
- nats-building: Add support for arm64

Fixes https://github.com/sunweaver/nextcloud-high-performance-backend-setup/issues/98
Adresses https://github.com/sunweaver/nextcloud-high-performance-backend-setup/issues/104 for arm64

### CODE REVIEW

File: src/setup-signaling.sh [hunk 1]
```diff
@@ -379,7 +385,13 @@ function signaling_step5() {
 	deploy_file "$TMP_DIR_PATH"/signaling/nginx-signaling-upstream-servers.conf /etc/nginx/snippets/signaling-upstream-servers.conf || true
 	deploy_file "$TMP_DIR_PATH"/signaling/nginx-signaling-forwarding.conf /etc/nginx/snippets/signaling-forwarding.conf || true
 
-	deploy_file "$TMP_DIR_PATH"/signaling/janus.jcfg /etc/janus/janus.jcfg || true
+	if [ "$(dpkg --print-architecture)" = "arm64" ]; then
+		deploy_file "$TMP_DIR_PATH"/signaling/janus_aarch64.jcfg /etc/janus/janus.jcfg || true
+	elif [ "$(dpkg --print-architecture)" = "ppc64el" ]; then
+		deploy_file "$TMP_DIR_PATH"/signaling/janus_powerpc64le.jcfg /etc/janus/janus.jcfg || true
```
Comment (sunweaver) [inline src/setup-signaling.sh:391]: Please remove the janus_powerpc64le.jcfg again, we don't have a use case for this as we don't support ppc64le as an architecture. Or do we?

**Review by sunweaver — COMMENTED** [review]
Please give feedback to my inline comment. Thanks.

File: src/setup-signaling.sh (hunk 1 above)
Comment (PhilDevProg) [reply src/setup-signaling.sh]: I removed support for PowerPC64LE

**Review by madscientist159 — COMMENTED** [review]
Why was ppc64le support removed?  We use this without issues on POWER servers.

**Comment by PhilDevProg** [comment]
> Why was ppc64le support removed?  We use this without issues on POWER servers.

I added it back but we don't have the hardware to test the script on ppc64le so we can't assure you that everything works or that everything will work in the future.

**Pull request merged**